packages/editor/src/client/blocks/mapLeafletBlock.tsx

```diff
@@ -22,148 +25,76 @@ Icon.Default.mergeOptions({
-export function MapLeafletItemElement({value, onChange}: FieldProps<MapLeafletItem>) {
-  const [isChooseModalOpen, setChooseModalOpen] = useState(false)
-  const [isEditModalOpen, setEditModalOpen] = useState(false)
-
-  //@ts-ignore
-  const {image, title, lat, lng, description} = value
-
-  return (
-    <>
-      <Box display="flex" flexDirection="row">
-        <Card
-          overflow="hidden"
-          width={200}
-          height={150}
-          marginRight={Spacing.ExtraSmall}
-          flexShrink={0}>
-          <PlaceholderInput onAddClick={() => setChooseModalOpen(true)}>
-            {image && (
-              <Box position="relative" width="100%" height="100%">
-                <Box position="absolute" zIndex={ZIndex.Default} right={0} top={0}>
-                  <IconButton
-                    icon={MaterialIconImageOutlined}
-                    title="Choose Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => setChooseModalOpen(true)}
-                  />
-                  <IconButton
-                    icon={MaterialIconEditOutlined}
-                    title="Edit Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => setEditModalOpen(true)}
-                  />
-                  <IconButton
-                    icon={MaterialIconClose}
-                    title="Remove Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => onChange(value => ({...value, image: null}))}
-                  />
-                </Box>
-                {image.previewURL && <Image src={image.previewURL} width="100%" height="100%" />}
-              </Box>
-            )}
-          </PlaceholderInput>
-        </Card>
-        <Box flexGrow={1}>
-          <TypographicTextArea
-            variant="h1"
-            placeholder="Title"
-            value={title}
-            onChange={e => {
-              const title = e.target.value
-              onChange(value => ({...value, title}))
+      <Box position="relative" width="100%" height="100%">
+        <Map
+          center={[centerLat, centerLng]}
+          zoom={zoom}
+          style={{width: '100%', height: '45vh'}}
+          onViewportChange={viewport => {
+            if (viewport.center && viewport.zoom) {
+              onChange({
+                ...value,
+                centerLat: viewport.center[0],
+                centerLng: viewport.center[1],
+                zoom: viewport.zoom
+              })
+            }
+          }}>
+          <TileLayer
+            url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+            attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+          />
+          {items.map((item, index) => (
+            <Marker position={[item.value.lat, item.value.lng]} key={index}>
+              <Popup>
+                <b>{item.value.title}</b> <br />
```

Comment:
i guess the title and the description should be paragraphs `<p>text</p>`

Reply:
done